### PR TITLE
test(collapsible): add initial test

### PR DESCRIPTION
**Pull request by devopsmariocom**

### CODE REVIEW

File: wct.conf.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+const packages = require( './package.json' );
```
Comment (hopkins-tk) [inline wct.conf.js]: Name  `packages` is a bit confusing (espetialy later on the `packages.packages` part) - please consider some better name:
- package
- packageJson
- packageInfo

**Review by hopkins-tk — CHANGES_REQUESTED** [review]

**Comment by devopsmariocom** [comment]
@hopkins-tk please review last changes, sorry had to rebase due to too many commits.

File: package.json [hunk 2]
```diff
@@ -1,22 +1,19 @@
...
-    "dist/polyfills.js",
-    "dist/vendors.js"
+    "dist"
   ],
   "scripts": {
-    "packagelist": "node utils/generate-packages-list.js",
+    "packagelist": "node toolbelt/generate-packages-list.js",
     "start": "npm run packagelist && webpack-dev-server --env.dev --hot --host 0.0.0.0",
-    "build": "npm run packagelist && rimraf dist && webpack --env.prod",
-    "build:publish": "npm run build && ./utils/deploy-ghpages.sh",
```
Comment (hopkins-tk) [inline package.json:17]: Won't we be missing this?
- I see the script stayed, but is nowhere used

File: wct.conf.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
+const packageJson = require( './package.json' );
+
+const config = {
+  "suites": packageJson.packages.map( ( package ) => `./packages/${package}/index.test.html` ),
+    "plugins": {
+    "sauce": {
```
Comment (hopkins-tk) [inline wct.conf.js:6]: Please indent :)

**Review by hopkins-tk — APPROVED** [review]

**Pull request merged**